### ISSUE TITLE
Move news to _data

### ISSUE THREAD

**Issue by miselico** [lr-vu/site#8]
Currently, news is directly collected into ./index.md in html form, making it difficult to modify and move later.

I suggest moving this to _data  in a yaml file, and then render it.

**Comment by pbloem** [comment]
Agreed. @tae898 Do you have time for this in the near future? I guess there's not a big hurry.

**Comment by tae898** [comment]
Yeah, I remember having a discussion about this with @pbloem some months ago. I did a bit of initial research, and somehow there wasn't really a straightforward way for me to achieve this. I might have missed something out. Also, if you have some pointers, please lemme know!
